cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -31,14 +31,14 @@ <h2 class="header-slug">
                 </h2>
             {% endif %}
             {% for post_type, post_type_list in related_post_types.iteritems() %}
-                {% set title, icon = (post_type, category_icon.render(post_type)) or ("Blog", "cf-icon-speech-bubble") %}
+                {% set title, icon = ( post_type, category_icon.render( post_type ) ) or ( "Blog", "cf-icon-speech-bubble" ) %}
```

Comment:
Not necessary to address here, but I think we need to remove `or ( "Blog",...` so that we don't show the speech bubble on categories without an associated icon. (Related to #1649)